CHANGELOG.md

```diff
@@ -4,6 +4,24 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 
+## [2.2.0] - 2021-07-06
+
+### Changed
+
+- `HttpClient` now does not create informational request logs by default.
```

Comment:
This is kinda breaking. I would rather leave the behavior unchanged and turn off the logging wherever needed

Reply:
It was my intention to make it breaking/not enabled by default since the logs were near useless in the previous format (didn't contain request body or query parameters).

By asking to explicitly enable it, my hope is that developers will need to make informed decisions when enabling such expensive logs. Happy to discuss this. Why you would rather leave it ON by default?

Reply:
My point was that you upgrade this library and your logs disappear. I'd rather upgrade the library & disable the logs to achieve this

Reply:
You're right. I was too much focused on what I want to do to (disable the logs) but from a library standpoint, such breaking change doesn't make sense (definitely not without bumping the major version). I will change it to leave the `requests` logs ON as they were before.

Reply:
Put back to be enabled by default.